### DISCUSSION TITLE
Welcome to Appium v2.0.0-beta.52

### DISCUSSION THREAD

**Discussion by rushian** [appium/appium#18097] [Release]
I want to share my commands to install and use Appium 2

I was hoping for a screen as it is in 1.22, but the server runs at the same prompt we call it.

First I uninstalled the previous version, with the command
`npm uninstall -g appium`
then I deleted the content from the directory C:\Users\YOUR_USERNAME\\.appium
The I execute each line above
```
npm install -g appium@next
appium driver install --source=npm appium-xcuitest-driver
appium driver install uiautomator2
appium driver update xcuitest
appium plugin install images
appium plugin install execute-driver
appium plugin install --source=npm appium-wait-plugin
appium plugin install --source=npm appium-dashboard
appium plugin install --source=npm appium-reporter-plugin
```
And to execute I use 
`appium server -ka 800 -pa /wd/hub --use-plugins="images,dashboard,wait-plugin,reporter-plugin,execute-driver"`

So I could start the session in Appium Inspector, using the following Remote Path: /

**Comment by jlipps** [comment]
did you delete the content at this directory:

```
C:\Users\YOUR_USERNAME.appium
```

or at this directory

```
C:\Users\YOUR_USERNAME\.appium
```

?

**Reply by rushian** [reply]
the second... i accidentally deleted

**Reply by jlipps** [reply]
ok good, that is what we expect. just wanting to make sure there was not some bug in where appium was writing content.

**Reply by rushian** [reply]
I edited the post...thanks
